src/interface/src/app/treatments/apply-treatment/apply-treatment.component.ts

```diff
@@ -70,6 +70,7 @@ export class ApplyTreatmentComponent {
     const action = this.prescriptionForm.get('prescriptionAction')?.value;
     if (action) {
       this.applyTreatments(action);
+      this.treatmentsState.loadSummaryDataOnly();
```

Comment:
i think this is wrong, this should be a side effect of updating the treated stands, no? So this line should be on `TreatmentsState.updateTreatedStands` , as a tap or map of `this.treatmentsService.setTreatments`, meaning we need to request summary data again AFTER we did the API call to update the treated stands.

Bummer that we need to refetch summary JUST to get the updated totalAcres.